contracts/interfaces/IDistributable.sol

```diff
@@ -3,10 +3,10 @@
 pragma solidity ^0.8.0;
 
 interface IDistributable {
-    error NotDistributor();
-    error NotProposedDistributor();
+    error NotDistributor(address caller);
```

Comment:
can you revert these changes? the new access control will deprecate these errors so less conflicts for the next PR